pygmt/tests/test_grdimage.py

```diff
@@ -154,7 +154,7 @@ def test_grdimage_central_meridians(grid, proj_type, lon0):
 # are still slightly different with an RMS error of 25, see issue at
 # https://github.com/GenericMappingTools/pygmt/issues/390
 # TO-DO remove tol=1.5 and pytest.mark.xfail once bug is solved in upstream GMT
-@check_figures_equal(tol=1.5)
+# @check_figures_equal(tol=1.5)
 @pytest.mark.parametrize("lat0", [0, 30])
 @pytest.mark.parametrize("lon0", [0, 123, 180])
 @pytest.mark.parametrize("proj_type", [pytest.param("Q", marks=pytest.mark.xfail), "S"])
```

Comment:
Why are the `test_grdimage_central_meridians_and_standard_parallels` all XPASS now?!! Arent we still using GMT 6.1.1? Does this mean #390 is magically resolved somehow?

```
..\tests\test_grdimage.py::test_grdimage_central_meridians_and_standard_parallels[Q-0-0] XPASS [ 53%]
..\tests\test_grdimage.py::test_grdimage_central_meridians_and_standard_parallels[Q-0-30] XPASS [ 53%]
..\tests\test_grdimage.py::test_grdimage_central_meridians_and_standard_parallels[Q-123-0] XPASS [ 54%]
..\tests\test_grdimage.py::test_grdimage_central_meridians_and_standard_parallels[Q-123-30] XPASS [ 54%]
..\tests\test_grdimage.py::test_grdimage_central_meridians_and_standard_parallels[Q-180-0] XPASS [ 54%]
..\tests\test_grdimage.py::test_grdimage_central_meridians_and_standard_parallels[Q-180-30] XPASS [ 55%]
```

Reply:
That's weird. The master branch CI says these tests are xfail:
```
../tests/test_grdimage.py::test_grdimage_central_meridians_and_standard_parallels[Q-123-0-png] XFAIL [ 52%]
../tests/test_grdimage.py::test_grdimage_central_meridians_and_standard_parallels[Q-123-30-png] XFAIL [ 53%]
../tests/test_grdimage.py::test_grdimage_central_meridians_and_standard_parallels[Q-180-0-png] XFAIL [ 53%]
../tests/test_grdimage.py::test_grdimage_central_meridians_and_standard_parallels[Q-180-30-png] XFAIL [ 53%]
```

Reply:
GMT Latest/master CI tests are all XPASS for those on Linux and macOS (see e.g. https://github.com/GenericMappingTools/pygmt/runs/1313952289#step:10:222) :confused: